### PR TITLE
fix: workaround dcm4che repo service disruption

### PR DESCRIPTION
**Pull request by mwithi**
Temporary workaround for dcm4che servers disruption.

These disruptions are frequent. OP-347 is aimed to permanently solve this kind of issues by getting rid of dcm4che libraries.

**NOTE**: delete/move previous downloaded libs in your .m2 local repository

### CODE REVIEW

**Comment by mwithi** [comment]
@dbmalkovsky can you please check this? Thanks!

**Comment by dbmalkovsky** [comment]
@mwithi I moved my local repository out of the way, pulled this PR, and then ran `mvn test` without issue so it looks good to me.  Thanks for doing this.

File: pom.xml [hunk 1]
```diff
@@ -425,17 +436,17 @@
 	<repositories>
 		<repository>
 			<id>lib</id>
-			<name>Legacy dependencies Repository</name>
-			<url>file:${project.basedir}/repo</url>
+			<name>Legacy Dependencies Repository</name>
+			<url>file:./repo</url>
```
Comment (dbmalkovsky) [inline pom.xml:440]: what was wrong with `${project.basedir}`?   It seems more robust and descriptive.

Comment (mwithi) [reply pom.xml]: I found that in some conditions it was creating the folder `${project.basedir}/`, thus I'm not able to reproduce it.

That's fine, I restore it and I also replace old `${basedir}` (deprecated)

**Review by emecas — COMMENTED** [review]
@mwithi ,  is the main idea of this PR keeping a local copy of dcm4che files under  /repo/dcm4che and removing `https://www.dcm4che.org/maven2` repo from pom.xml ?

**Comment by mwithi** [comment]
> @mwithi , is the main idea of this PR keeping a local copy of dcm4che files under /repo/dcm4che and removing `https://www.dcm4che.org/maven2` repo from pom.xml ?

yes

**Review by emecas — APPROVED** [review]

**Pull request merged**